search-update.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: July 15, 2019
+   :tags: gsoc, search, feature, in-doc-search
+   :author: Vaibhav
+   :location: LKO
+
+Search Update
+=============
+
+Giving users the ability to easily find the information that they
+are looking for has always been important for Read the Docs.
+This year, I, `Vaibhav Gupta`_, took the opportunity provided
+by Google Summer of Code to improve the search.
+The main goal of my `GSoC project`_ was to enable multiple hits per search result,
+improve the UI/UX around code search and to make an `Sphinx`_ extension to provide
```

Comment:
I would link this directly to the Sphinx extension here rather than to Sphinx itself.